docs/user-manual/en/amqp-broker-connections.md

```diff
@@ -0,0 +1,238 @@
+
+To define an AMQP broker connection, add an `<amqp-connection>` element within the `<broker-connections` element in the `broker.xml` configuration file. For example:
+
+```xml
+<broker-connections>
+    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker" retry-interval="100" reconnect-attempts="-1" user="john" password="doe">
+         ...
+    </amqp-connection>
+</broker-connections>
+```
+
+- `uri`: tcp://host:myport (this is a required argument)
+- `name`: Name of the connection used for management purposes
+- `user`: User name with which to connect to the endpoint (this is an optional argument)
+- `password`: Password with which to connect to the endpoint (this is an optional argument)
+- `retry-interval`: Time, in milliseconds to wait before retrying a connection after an error. The default value is `5000`.
+- `reconnect-attempts`: default is -1 meaning infinite
+- `auto-start` : Should the broker connection start automatically with the broker. Default is `true`. If false you need to call a management operation to start it.
+
+*Notice*: If you disable auto-start on the broker connection, the start of the broker connection will only happen after the management method `startBrokerConnection(connectionName)` is called on the ServerController.
+
+*Important*: The target endpoint needs permission for all operations that you configure. Therefore, If you are using a security manager, ensure that you perform the configured operations as a user with sufficient permissions.
+
+# AMQP Server Connection Operations
+The following types of operations are supported on a AMQP server connection:
+
+* Senders
+    * Messages received on specific queues are transferred to another endpoint
+* Receivers
+    * The broker pulls messages from another endpoint
+* Peers
+    * The broker creates both senders and receivers on another endpoint that knows how to handle them. Currently, this is implemented by Apache Qpid Dispatch.
+* Mirrors
+    * The broker uses an AMQP connection to another broker and duplicate messages and sends acknowledgements over the wire.
+
+## Senders and Receivers
+It is possible to connect an ActiveMQ Artemis broker to another AMQP endpoint simply by creating a sender or receiver broker connection element.
+
+For a `sender`, the broker creates a message consumer on a queue that sends messages to another AMQP endpoint.
+
+For a `receiver`, the broker creates a message producer on an address that receives messages from another AMQP endpoint.
+
+Both elements work like a message bridge. However, there is no additional overhead required to process messages. Senders and receivers behave just like any other consumer or producer in ActiveMQ Artemis.
+
+You can configure senders or receivers for specific queues. You can also match senders and receivers to specific addresses or _sets_ of addresses, using wildcard expressions. When configuring a sender or receiver, you can set the following properties:
```

Comment:
the broker currently does not define matching for queues.. just for addresses. I"m keeping it consistent with everything else.

Reply:
Ok. I think its rather inconsistent with itself though. Why not 'match' and 'address' for example rather than 'match' and 'queue'?

Continuing the second bit of my earlier comment, I actually think this has confused me on what these features as a whole even actually do exactly. I've been reviewing it for a week and I'm still unclear on the expected behaviours, even after its now been merged. Which is why I keep banging on about the examples and docs clearly outlining what the expected behaviour is.  I'm actually starting to think some of it may be doing the wrong thing. But im still not sure.